definitions/procedures/pulp/remove.rb

```diff
@@ -24,7 +24,18 @@ def pulp_data_dirs
         '/var/lib/pulp/uploads',
         '/var/lib/mongodb/',
         '/var/cache/pulp'
-      ]
+      ].select { |dir| File.directory?(dir) }
+    end
+
+    def pulp_data_dirs_mountpoints
+      pulp_data_dirs.map do |pulp_dir|
+        dir_path = Pathname.new(pulp_dir)
+        pulp_dir if dir_path.mountpoint?
+      end.compact
+    end
```

Comment:
```suggestion
    def pulp_data_dirs_mountpoints
      pulp_data_dirs.select{ |d| Pathname(d).mountpoint? }.compact
    end
```

Reply:
And I think you need to `require 'pathname'` for this to work?

Reply:
Yeah, was about the require it, however I can see installer feature is using it already, I can't find explicit require anywhere. But yes it works without require.